### PR TITLE
Lonesome Donator Piece Test

### PR DESCRIPTION
**Pull request by Wehraboo**
Basically what the title says, my first go at this.

- Added ODST donator armour "wehraboo", wasn't quite sure if I needed to make it a lonesome dmi or not, so I just threw it into the ODSTDonorPackage. 

- Edited the ODST config file to include said donator sprite. 

Should be able to help other donators if this goes well.

### CODE REVIEW

File: code/modules/halo/clothing/odst.dm [hunk 1]
```diff
@@ -488,6 +488,30 @@
 
 	icon_state = "Odst Armor Maxattacker"
 
+/obj/item/clothing/head/helmet/odst/donator/wehraboo
```
Comment (caelaislinn) [inline code/modules/halo/clothing/odst.dm:491]: I'm assuming the sprites are in ODSTDonorPackage.dmi? You'll need to set icon = that for the sprites to be correctly set

File: code/modules/halo/clothing/odst.dm [hunk 2]
```diff
@@ -488,6 +488,30 @@
...
+	if (on)
+		item_state = "wehraboo-helmet_worn"
+		icon_state = "wehraboo-helmet_obj"
+	else
+		item_state = "wehraboo-helmet_worn"
+		icon_state = "wehraboo-helmet_obj"
+		update_clothing_icon()
+	. = ..()
+
+/obj/item/clothing/suit/armor/special/odst/donator/wehraboo
```
Comment (caelaislinn) [inline code/modules/halo/clothing/odst.dm:511]: set icon here as well

**Review by caelaislinn — CHANGES_REQUESTED** [review]

File: code/modules/halo/clothing/odst.dm (hunk 1 above)
Comment (Wehraboo) [reply code/modules/halo/clothing/odst.dm:491]: Little unsure how to correctly reference the right .dmi, looked at the other donator sets as an example but I'm still clueless on it.

**Comment by daPotato** [comment]
Your helmet doesn't have a transparent state, my brain organ tells me the Toggle_VISR01 blah blah code is unneeded, Cael can confirm me on this, I don't understand the way it handles visor switch states and I'm just here because I am bored.

I thiiiink Cael's change requests should push the fixes to the issues to your PR, but I don't know how GitHub works so don't quote me on that either.

File: code/modules/halo/clothing/odst.dm (hunk 1 above)
Comment (caelaislinn) [reply code/modules/halo/clothing/odst.dm:491]: I double checked the code and what you've done here is actually correct.

**Review by caelaislinn — APPROVED** [review]

File: code/modules/halo/clothing/odst.dm [hunk 3]
```diff
@@ -488,6 +488,30 @@
...
+	on = !on
+	update_icon()
+	if (on)
+		item_state = "wehraboo-helmet_worn"
+		icon_state = "wehraboo-helmet_obj"
+	else
+		item_state = "wehraboo-helmet_worn"
+		icon_state = "wehraboo-helmet_obj"
+		update_clothing_icon()
+	. = ..()
```
Comment (caelaislinn) [inline code/modules/halo/clothing/odst.dm:509]: Potato is right, there is no sprite for when the visor is open. This is all fine and I can merge as is if you want, but toggling the visor will not have any visual effect.

Comment (Wehraboo) [reply code/modules/halo/clothing/odst.dm:509]: Yeah, this particular bunch of sprites doesn't have an open variant, which's fine with me. Merge whenever, thanks for checking this out.

Comment (caelaislinn) [reply code/modules/halo/clothing/odst.dm:509]: The visor toggle button is dead code then. I'll merge this now but come back and clean it up later; I think we still have some ODST donators outstanding

**Pull request merged**